eslint/Tests/EslintUnitTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\User\Block\Role\Tab;
+
+use function PHPUnit\Framework\assertStringContainsString;
+
+class EslintUnitTest extends \PHPUnit\Framework\TestCase
+{
+    const command = '../node_modules/.bin/eslint -c eslint/.eslintrc --rulesdir eslint/rules ';
```

Comment:
Do not forget to specify access level
```suggestion
    private const command = '../node_modules/.bin/eslint -c eslint/.eslintrc --rulesdir eslint/rules ';
```